### PR TITLE
Added general info for OpenOS and Lua programming, added text to index

### PR DESCRIPTION
**Pull request by rashdanml**
Stuffs. The Lua programming doc can probably expanded to include more examples on how to use different components, maybe even add a file summarizing the custom libraries available (basic info, "this library does this, that, and something else"). Would link up to component API docs (and other API docs) if those get added in the future.

### CODE REVIEW

File: src/main/resources/assets/opencomputers/doc/en_US/general/lua.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Lua 
+
+The Lua [reference manual](www.lua.org/manual/5.2/manual.html) is a good place to get started with the basics of Lua and becoming familiar with the basic syntax and standard libraries. OpenComputers has very similar standard libraries, and omits certain libraries entirely (such as the debug library).
```
Comment (fnuecke) [inline src/main/resources/assets/opencomputers/doc/en_US/general/lua.md]: External links have to be prefixed with `http://` or `https://` to be recognized.

Minor correction: debug library is missing 99%, but traceback is available.

File: src/main/resources/assets/opencomputers/doc/en_US/general/openOS.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# OpenOS
+
+OpenOS is a basic operating system available in OpenComputers. It is required to boot up a [computer](computer.md) for the first time, and can be crafted by placing an empty [floppy disk](../item/floppy.md) and a Minecraft book inside a Crafting table.
```
Comment (fnuecke) [inline src/main/resources/assets/opencomputers/doc/en_US/general/openOS.md]: Will be floppy and manual now.

File: src/main/resources/assets/opencomputers/doc/en_US/general/openOS.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+Once crafted, the [floppy disk](../item/floppy.md) can be placed inside a [disk drive](../block/diskDrive.md) connected to a [computer](computer.md) system, which will allow the [computer](computer.md) to boot up OpenOS. Once booted, it is advisable to install OpenOS to an empty [hard drive](../item/hdd1.md), foregoing the need for a [floppy disk](../item/floppy.md). A tier 3 [computer case](../block/case3.md) does not require a [disk drive](../block/diskDrive.md), as it has a slot built in for the [floppy disk](../item/floppy.md). OpenOS can be installed by simply running *install*, and following the on-screen prompts to complete the installation. The [floppy disk](../item/floppy.md) may be removed once the system has been rebooted. OpenOS can be installed on all devices except [drones](../item/drone.md) and [microcontrollers](../block/microcontroller.md) (both of which require manually programming an [EEPROM](../item/eeprom.md) to provide functionality).
```
Comment (fnuecke) [inline src/main/resources/assets/opencomputers/doc/en_US/general/openOS.md]: Also, the more important reason to install on a disk: read-write file system. Can't save things on the OpenOS floppy.

File: src/main/resources/assets/opencomputers/doc/en_US/index.md [hunk 4]
```diff
@@ -1,12 +1,32 @@
 # OpenComputers Manual
 
-This will be the index, one glorious day...
+OpenComputers is a mod that adds persistent, modular, and highly configurable [computers](general/computer.md), [servers](item/server1.md), robots, and drones. All devices can be programmed using Lua 5.2, allowing for systems with varying complexity depending on the usage.
```
Comment (fnuecke) [inline src/main/resources/assets/opencomputers/doc/en_US/index.md]: Also link robots and drones?

**Comment by fnuecke** [comment]
Awesome, thanks a lot! Really like it. Had a few notes, but mostly minor nitpicks :-)

**Comment by rashdanml** [comment]
Will make changes, cheers. 

Note to self: in future, make changes locally and push as one commit >.> Sorry for the multiple commits XD

**Comment by KJA1582** [comment]
Yeah, I have similar problems with commits. Just squash your commits into one (best google how to do so).

**Comment by rashdanml** [comment]
I dun goofed. Trying to squash my two commits into one somehow managed to mess with a bunch of other files D:

**Comment by evieluvsrainbows** [comment]
@rashdanml Yeah, I knew something went wrong when I saw the "This pull request contains merge conflicts that must be resolved." message and the 14 files you "changed"... :D

**Comment by rashdanml** [comment]
I suppose it would be easier to close this PR and start with a fresh fork (have the changes made locally).

**Pull request closed without merging**